app/assets/stylesheets/exam_templates.scss

```diff
@@ -120,3 +120,39 @@
   color: #8d8d8d !important;
   font-style: italic;
 }
+
+#log_entries {
+  width: 100%;
+
+  td, th {
+    padding: 20px 15px;
+    text-align: right;
+  }
+
+  th {
+    cursor: pointer;
+  }
+
+  td:nth-child(-n+4), th:nth-child(-n+4){
+    text-align: left;
+  }
+
+  thead {
+    border-bottom: 1px solid $grey;
+  }
+
+  tbody {
+
+    td:first-of-type {
```

Comment:
I'm not keen on default grey highlighting; I'd rather just have these white. The yellow highlighting strikes me as a bit too bold. Instead, I would like you to use the warning icon (you can see this in the submissions table) and stick it next to the error number in the last column.